ui/src/actions/actions-ui.js

```diff
@@ -12,7 +12,7 @@ export const appsTableSort = (_event, index, direction) => {
   };
 };
 
-export const appDetailsSort = (_event, index, direction) => {
+export const appVersionsTableSort = (_event, index, direction) => {
```

Comment:
does `_event` need to be passed in here?

Reply:
I would imagine they do need to be @damienomurchu - at least with how it is currently implemented.

The action is being called directly from the Patternfly table, and `_event` is an argument that comes from the table. So if it were to be removed, `index` would become `_event`.

https://github.com/aerogear/mobile-security-service/blob/686a49a8edb8868555d37477145ea1eeef5abcbb/ui/src/containers/appView/AppVersionsTableContainer.js#L85

Reply:
The alternative is to have another function and only pass in the index and direction.

Reply:
yes, that was what I was thinking @davidffrench, as at the moment the reducer is being called directly instead of via a handler

Reply:
Just less code to call the action creators directly rather than have an intermediate function.